test/cmdlineTests.sh

```diff
@@ -103,17 +124,22 @@ function test_solc_behaviour()
     local stdout_expected="${4}"
     local exit_code_expected="${5}"
     local stderr_expected="${6}"
+    local stdout_expectation_file="${7}" # the file to write to when user chooses to update stdout expectation
+    local stderr_expectation_file="${8}" # the file to write to when user chooses to update stderr expectation
     local stdout_path=`mktemp`
     local stderr_path=`mktemp`
+
+    trap "rm -f $stdout_path $stderr_path" EXIT
+
     if [[ "$exit_code_expected" = "" ]]; then exit_code_expected="0"; fi
 
+    local solc_command="$SOLC ${filename} ${solc_args}"
+    if [[ -n "$solc_stdin"  ]]; then solc_command+=" <$solc_stdin"  ; fi
```

Comment:
Is it enough to quote `solc_stdin` in order to not make this script vulnerable in the context of `eval` below?

Reply:
Did you mean that someone will create some directories with strange directory name to execute malicious shell command on solidity developer's computer?

Reply:
(Asked the above question because I couldn't catch what you said)

Reply:
Yes, I wasn't thinking about weird directory names in particular, but rather about a weird standard input. But that should not be problem, since it's only passed internally. As far as I can see, the filenames etc. are properly quoted. So it should be fine.